app/components/nav-dropdown.tsx

```diff
@@ -0,0 +1,33 @@
+import { ChevronDown } from "lucide-react";
+import Link from "next/link";
+
+export default function PastYearsDropdown() {
+  const menuItems = [
+    // { label: "2024", href: "/projects" },
```

Comment:
I realized that it can be confusing if we don't include 2024 in the list, so please put this in. 😅

Reply:
One last thing @dotnize. Thank you so much for your PR! 😊

Reply:
yup haha, should be good now! let me know if you need any more changes :)